ValeStyles/Couchbase/Quotations.yml

```diff
@@ -8,6 +8,4 @@ tokens:
   - '"[^"]*"'
 
 exceptions: 
-  - \x60.+".+".+\x60
-  - \x60.+".+"\x60
-  - \x60".+"\x60
\ No newline at end of file
+  - '\x60"([^"]*)"\x60'
```

Comment:
This will now only match:

 backtick + quote + any number of non-quotes, + quote + backtick?

Is that what's intended? Didn't you want `.*` before and after to allow tokens before/after the quoted sections?

Reply:
Hmm, good point. Let's add that back in, just in case.